examples/google-esp-integration/esp-integration.md

```diff
@@ -0,0 +1,47 @@
+[UID2 Overview](../../../README.md) > [Getting Started](../../README.md) > [v2](../README.md) > [Integration Guides](README.md) > UID2 Secure Signal Integration Guide
+
+# UID2 Secure Signal Integration Guide
+
+This guide covers integration steps for organizations using UID2 with Google Secure Signal (GSS, previously known as Google Encrypted Signal, ESP). It includes the following sections:
+
+* [Overview](#overview)
+* [Allow Secure Signal Sharing](#allow-secure-signal-sharing)
+* [Publisher Integrations](#publisher-integration)
+* [Server-Only Integration](#server-only-integration)
+* [UID2 SDK Integration](#uid2-sdk-integration)
+
+>NOTE: 
+To use UID2 Secure Signal integration, you must be using the UID2 JavaScript SDK.
```

Comment:
@genwhittTTD this is optional, they could choose to use server-only integration, which UID2 SDK is not required.